package_control/commands/satisfy_dependencies_command.py

```diff
@@ -58,3 +61,7 @@ def run(self):
                 Please check the console for details.
                 '''
             )
+            error = True
+
+        if not error:
+            console_write(u"All dependencies have been satisfied.")
```

Comment:
For simple messages, the currently style is to omit the `.`. If more than one sentence is required a `.` is included.